main.cpp

```diff
@@ -686,15 +686,23 @@ void PrintAppInfo()
 		"; Author: Antonio Maiorano (amaiorano at gmail dot com)\n"
 		"; Source code available at http://github.com/amaiorano/nes-disasm/\n"
 		"\n";
-
+#ifdef _WIN32
```

Comment:
You know what, don't bother keeping the old code. Your changes for Linux are valid for MSVC builds as well. Just replace with `printf("%s", text);`